### PR TITLE
Add player control keybinds

### LINKED ISSUE #112: Add current skip time selection to seek bar
When creating a skip time, the current start and end time should appear in the seek bar of the video player.

### LINKED ISSUE #113: Add video controls keybinds
Add keybinds for moving the current time forwards and backwards by one frame like on YouTube. The default keybinds are `,` and `.`. We can also add keybinds to manually skip an opening / ending (+-90s to the current time).

### LINKED ISSUE #115: Add `zoro` page support
Website: https://zoro.to/
Player: Plyr, JW
Language: English

### PR DESCRIPTION
**Pull request by lexesjan**
## Purpose

There are some useful keybinds which do not exist in all players which is useful for creating skip times. Closes #112, closes #113, closes #115.

## Proposed Change

Add keybinds to seek forward or backward by one frame. Add keybinds to move the current time forward and backward by the skip time length (90s by default).

## Checklist

- [x] Add frame by frame video seeking keybinds
- [x] Add seek forward and backward keybind
- [x] Bump version minor
- [ ] ~~Obtain video framerate instead of hard-coded frame rate~~ Not currently possible

## Additional

Out of scope changes:
- [x] Add preview skip time to seek bar
- [x] Blur background for submit and vote menu
- [x] Fix add keybind style on small screens
- [x] Change preview button behaviour to go to end time if it is selected
- [x] Add zoro page support

### CODE REVIEW

File: src/pages/zoro/zoro.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { ZoroSyncData } from './zoro.types';
+import metadata from './metadata.json';
+
+export class Zoro extends BasePage {
+  static getMetadata(): Metadata {
+    return metadata;
+  }
+
+  getSyncData(): ZoroSyncData {
+    return JSON.parse(document.getElementById('syncData')?.textContent ?? '{}');
```
Comment (dabreadman) [inline src/pages/zoro/zoro.ts:13]: Oww, handy.
Actually how do you go on doing this? They seem to redirect you to home page when you open debugger.

Comment (lexesjan) [reply src/pages/zoro/zoro.ts:13]: Add `zoro.to##+js(aopr, console.clear)` to your ublock origin filters.

It comes from https://github.com/Karmesinrot/Anifiltrs

(Tip from henrik9999)

File: src/scripts/background/types.ts [hunk 2]
```diff
@@ -25,19 +21,17 @@ export const DEFAULT_SKIP_OPTIONS: SkipOptions = {
   recap: 'manual-skip',
 } as const;
 
-export type SkipTimeIndicatorColours = Omit<
-  {
-    [T in SkipType]: string;
-  },
-  'preview'
->;
+export type SkipTimeIndicatorColours = {
```
Comment (dabreadman) [inline src/scripts/background/types.ts:24]: colours :?

File: src/scripts/background/types.ts [hunk 3]
```diff
@@ -71,6 +78,8 @@ export const DEFAULT_KEYBINDS: Keybinds = {
   'increase-current-time-large': 'Shift+L',
   'decrease-current-time': 'j',
   'decrease-current-time-large': 'Shift+J',
+  'seek-forward-one-frame': '.',
```
Comment (dabreadman) [inline src/scripts/background/types.ts:81]: `Ctrl` + `Alt` + `->`
`Ctrl` + `Alt` + `<-`
?

**Review by dabreadman — APPROVED** [review]
LGTM, might be time to #119 #120

File: src/scripts/background/types.ts (hunk 3 above)
Comment (lexesjan) [reply src/scripts/background/types.ts:81]: Might leave it as is as it's the default YouTube keybinds

File: src/scripts/background/types.ts (hunk 2 above)
Comment (lexesjan) [reply src/scripts/background/types.ts:24]: Its the type for the skip time indicator dictionary which maps a skip type to a colour

**Comment by lexesjan** [comment]
> LGTM, might be time to #119 #120

Agreed

**Pull request merged**